cts/cts-cli.in

```diff
@@ -853,6 +856,63 @@ EOF
     rm -f "$TMPXML"
 }
 
+test_rules() {
+    local TMPXML
+
+    export CIB_shadow_dir="${shadow_dir}"
+    $VALGRIND_CMD crm_shadow --batch --force --create-empty $shadow --validate-with pacemaker-2.10 2>&1
```

Comment:
We don't need to specify --validate-with here, it'll use the latest by default. (It makes sense to specify it in the validity and upgrade tests, because they're specifically testing the ability to handle older schemas. I'm not sure why it's in the acls test.)